src/web/routes/application/address/manual-address/maunal-address.test.js

```diff
@@ -62,6 +62,15 @@ test('Address contentSummary() should return content summary in correct format w
   t.end()
 })
 
+test('Address contentSummary() is null when there is a selected address on the session', (t) => {
+  const testReq = assocPath(['session', 'claim', 'selectedAddress'], 'test address', req)
```

Comment:
Could we just have this as JSON rather than using assocPath for clarity? We do this in other unit tests...

Reply:
This is consistent with the rest of the test case. If we make this change it should be it's own PR due to the size of the change.